opennms-base-assembly/src/main/filtered/etc/collectd-configuration.xml

```diff
@@ -109,10 +109,30 @@
          <parameter key="retry" value="2"/>
          <parameter key="timeout" value="3000"/>
          <parameter key="rrd-base-name" value="java"/>
-         <parameter key="collection" value="jsr160"/>
+         <parameter key="collection" value="jmx-opennms"/>
```

Comment:
We have to ensure that this does not break existing behaviour, e.g. does with this change the `JMXDetector` still work for OpenNMS?

Reply:
> We have to ensure that this does not break existing behavior, e.g. does with this change the `JMXDetector` still work for OpenNMS?

yes, this is the reference to the data collection package in collectd and does not interfere with the JMX Detector.

Reply:
That was probably a bad example. I found other references to "jsr160" in the code, e.g. in https://github.com/OpenNMS/opennms/blob/9eaac5f862c6c8891bb63326b84a7f6e946d451d/opennms-doc/guide-admin/src/asciidoc/text/performance-data-collection/collectors/jmx.adoc#L150 or some data-collection examples. We should make them work as well, if we change the value here, that's all I am saying :)